### ISSUE TITLE
problems with translation

### ISSUE THREAD

**Issue by sonaldo** [MonitorControl/MonitorControl#1199]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Only the menus are translated, the settings remain in English.
<img width="725" alt="image" src="https://user-images.githubusercontent.com/3761019/194319967-5386fe54-c0a7-41db-9c0a-0b6a84a7eed3.png">

### Steps to reproduce

1. Go to 'Preferences'

### Expected behavior

All options in the system language, in this case pt-br

### Anything else?

<img width="727" alt="image" src="https://user-images.githubusercontent.com/3761019/194320124-68601e3b-6ea9-46ea-b716-1ab42a9f86d3.png">

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.6
- Mac model: Macbook Air 2020
- MonitorControl version: 4.1.0
- Monitor(s): LG HDR 4k
- Apple Silicon/M1 (yes or no): Yes
```

**Comment by x-magic** [comment]
It seems brazilian portuguese is a supported language: https://github.com/MonitorControl/MonitorControl/tree/main/MonitorControl/UI/pt-BR.lproj

Maybe try to [force the app language in system preferences](https://support.apple.com/en-au/guide/mac-help/mh26684/mac#:~:text=Choose%20the%20language%20you%20use%20for%20individual%20apps) and see if the app language changes. Otherwise it could be the app's problem.

**Comment by waydabber** [comment]
Yes, pt-BR should be supported. Is this still an issue? If not, it should be closed. :)
